src/Uno.UI/UI/Xaml/Style/Generic/Generic.xaml

```diff
@@ -221,10 +221,6 @@
 
 	<Style x:Key="XamlDefaultTextBox"
 		   TargetType="TextBox">
-		<Setter Property="MinWidth"
```

Comment:
In `controls/dev/CommonStyles/TextBox_themeresources.xaml` from WinUI code, I see:

```xaml
<Setter Property="MinHeight" Value="{ThemeResource TextControlThemeMinHeight}" />
<Setter Property="MinWidth" Value="{ThemeResource TextControlThemeMinWidth}" />
```

Reply:
Maybe there is something about `ComboBoxTextBoxStyle` not being used properly or the style itself having an issue?

Reply:
Reading the tests, I see the intent is that this is only for old UWP styles. If that's the intent, I think it's right and the comment can be resolved

Reply:
yes indeed, the fluent styles have the properties explicit, but the ye old generic "UWP" styles don't 😂 . It is quite odd!